sql/core/src/main/scala/org/apache/spark/sql/execution/ExistingRDD.scala

```diff
@@ -103,6 +103,8 @@ case class ExternalRDDScanExec[T](
   override lazy val metrics = Map(
     "numOutputRows" -> SQLMetrics.createMetric(sparkContext, "number of output rows"))
 
+  override val nodeName: String = s"Scan ExternalRDD ${output.map(_.name).mkString("[", ",", "]")}"
```

Comment:
I don't think including the output in the node name is a good idea.

Reply:
My intention here was to be able to distinguish between `ExternalRDDScanExec` nodes. If we remove the `output` part from `nodename`, then these nodes would be named as `Scan ExternalRDD` which is generic.